### PR TITLE
Fix up and reorganize README

### PR DESCRIPTION
**Pull request by tiswanso**
update the readme to reflect latest install via Istio helm, troubleshooting info, etc.  General cleanup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,6 +76,50 @@ $ oc adm policy add-scc-to-user privileged -z istio-cni -n kube-system
...
+
+1. clone this repo
+
+1. Install Istio control-plane
+
+1. Create Istio CNI installation manifest--either manually or via Helm:
+
+   1. (Helm Option) Construct a `helm template` or `helm install` command for your Kubernetes environment
+   
+      ```sh
```
Comment (venilnoronha) [inline README.md]: \`\`\`sh -> \`\`\`console, here and below.

File: README.md [hunk 2]
```diff
@@ -219,6 +192,47 @@ To make use of the `istio-cni` chart from another chart:
 
 The Istio CNI testing strategy and execution details are explained [here](test/README.md).
 
+## Troubleshooting
+
+### Validate the iptables are modified
+
+1. Collect your pod's container id using kubectl.
+```sh
```
Comment (venilnoronha) [inline README.md]: nit: add line break before code block

File: README.md [hunk 3]
```diff
@@ -219,6 +192,47 @@ To make use of the `istio-cni` chart from another chart:
...
+$ ns=test-istio
+$ podnm=reviews-v1-6b7f6db5c5-59jhf
+$ container_id=$(kubectl get pod -n ${ns} ${podnm} -o jsonpath="{.status.containerStatuses[?(@.name=='istio-proxy')].containerID}" | sed -n 's/docker:\/\/\(.*\)/\1/p')
+```
+
+2. SSH into the Kubernetes' worker node that runs your pod.
```
Comment (venilnoronha) [inline README.md]: nit: Kubernetes' -> Kubernetes

File: README.md [hunk 4]
```diff
@@ -294,7 +308,8 @@ Workflow:
     1.  determine containerPort list
 1.  Determine if the pod needs to be setup for Istio sidecar proxy
     1.  if pod has a container named `istio-proxy` AND pod has more than 1 container
-        1.  Final Logic TBD -- e.g. pod labels?  namespace checks?
+        1.  if pod has annotation with key `sidecar.istio.io/inject` with value `false` then skip redirect
+        1.  else do redirect
```
Comment (venilnoronha) [inline README.md]: nit: capitalize first character in both points.

File: README.md [hunk 5]
```diff
@@ -11,88 +11,28 @@ The CNI handling the netns setup replaces the current Istio approach using a `NE
...
-         |-------------|-------------|
-         | default kubeadm | [values.yaml](deployments/kubernetes/install/helm/istio-cni/values.yaml) |
-         | GKE | [values_gke.yaml](deployments/kubernetes/install/helm/istio-cni/values_gke.yaml) |
+| Option | Values | Default | Description |
+|--------|--------|---------|-------------|
+| hub | | | The container registry to pull the `install-cni` image. |
+| tag | | | The container tag to use to pull the `install-cni` image. |
+| logLevel | `panic`, `fatal`, `error`, `warn`, `info`, `debug` | `warn` | Logging level for CNI binary |
+| excludeNamespaces | `[]string` | `[ istio-system ]` | List of namespaces to exclude from Istio pod check |
+| cniBinDir | | `/opt/cni/bin` | Must be the same as the environment's `--cni-bin-dir` setting (kubelet param) |
```
[the remark this answers is not in the document]
Comment (tiswanso) [reply README.md]: replaced globally

File: README.md (hunk 1 above)
Comment (tiswanso) [reply README.md]: global replaced

File: README.md (hunk 2 above)
Comment (tiswanso) [reply README.md]: I think I got all these

File: README.md (hunk 4 above)
Comment (tiswanso) [reply README.md]: fixed other points around this one

**Comment by tiswanso** [comment]
@venilnoronha 
thanks for the quick review!

**Pull request merged**